data/campaigns/Northern_Rebirth/scenarios/02_01_Infested_Caves.cfg

```diff
@@ -543,7 +543,7 @@
             [then]
                 [message]
                     speaker=unit
-                    message= _ "Hey look, it’s an ancient door. It seems the trolls were too stupid to find how to open it."
+                    message= _ "Hey look, it’s an ancient door. It seems the trolls were too stupid to figure out how to open it."
```

Comment:
The original reads fine to me. But this brings up an interesting example - if we're going to have a separate en_US localisation, this change belongs in that localisation.

Reply:
That's a good point. Do you think it's not a change worth having then? I like the revised one more but it might not be worth annoying translators?

Reply:
I'm not sure I'm the one to make the decision on it. I just know that with this change I'll be adapting it for en_GB.

Reply:
Yes, we want to avoid modern words and phrases whenever possible but considering our audience is modern readers there is some leeway for clarity. NR was on the side of casualness to begin with as well which makes it more suitable for more modern text.

I'm just asking for other opinions, I don't mind requesting that the original be kept if others feel that it makes undue work for translation.

Reply:
If en_us isn't the writing language we're using for the scenarios, what are we going to use welsh?

In fiction writing, you can get it to sound old without making it sound like Shakespeare. Because while Shakespeare is good, it's hard to read.

Also, if we're trying to not use modern words, there is a lot of work in later scenarios. Because in scenario 5 there are statements like: "Geez, what a nutcase." and "Holy Cow!" Those are not old-timey sounding.

Reply:
We will keep those phrases for now. NR is one of the more casual style campaigns which is why I don't mind the more modern language. I do want some extra feedback on the translation stuff but not sure if others will respond. Will leave open a few more days for discussion on that but otherwise it looks done for me.

Reply:
Ok. I'll make a branch and work on the next campaign scenario, so it's ready to go.